docs/100-upgrade/migration-guides/3.3-3.4.mdx

```diff
@@ -65,3 +65,67 @@ normal commands used in `remix`.
   }
 }
 ```
+
+### Removal of `Meta Modules`
+
+In this version we have removed support for
+[`Meta Modules`](https://developers.front-commerce.com/docs/2.x/advanced/graphql/meta-modules#how-to-define-a-metamodule).
+This will be handled by the automated migration script.
```

Comment:
**nitpick:** I think it's important to explain why, and what is the alternative.

**suggestion:** 
```suggestion
In this version we have removed support for
[`Meta Modules`](https://developers.front-commerce.com/docs/2.x/advanced/graphql/meta-modules#how-to-define-a-metamodule) from the final GraphQL module definition API.
Front-Commerce extensions shared the same responsibility, and should be used instead.
This will be handled by the automated migration script.
```